### PR TITLE
Suggestions for Lesson 16

### PR DESCRIPTION
**Pull request by olof-nord**
* Adopted from the spring lesson, but offering more topics
* Quick review on the newLesson script
* Address typo from previous lesson

### CODE REVIEW

File: lesson16.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+https://date.nager.at/Api
+> Query the holidays of over 100 countries
+
+---
+
+### Task
+
+Show the holidays in Germany for 2024.
+This differs from state to state. Add a dropdown to select a specific state?
```
Comment (haraldF) [inline lesson16.md]: I wouldn't phrase it as a question. Maybe as a bonus task, like:

BONUS: Add a dropdown to select holidays for one state only.

File: lesson16.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+---
+
+## Maps
+
+https://leafletjs.com/examples/quick-start/
```
Comment (haraldF) [inline lesson16.md:50]: Doesn't this require an API key?

**Review by haraldF — APPROVED** [review]
Nice ideas! I suggest that students choose one, max 2 projects to work on. Maybe we can then create one breakout room per project?

File: lesson16.md (hunk 2 above)
Comment (olof-nord) [reply lesson16.md:50]: Maybe it did in the past, but not anymore.
here a small Proof of Concept, basically copying and pasting the code from the quick start tutorial:

```html
<html lang="en">
<head>
  <title>Leaflet Demo</title>
  <link rel="stylesheet" href="https://unpkg.com/leaflet@1.9.4/dist/leaflet.css"
        integrity="sha256-p4NxAoJBhIIN+hmNHrzRCf9tD/miZyoHS5obTRR9BMY="
        crossorigin="" />
  <!-- Make sure you put this AFTER Leaflet's CSS -->
  <script src="https://unpkg.com/leaflet@1.9.4/dist/leaflet.js"
          integrity="sha256-20nQCchB9co0qIjJZRGuk2/Z9VM+kNiyxNV1lvTlZBo="
          crossorigin=""></script>
  <style>
      #map {
          height: 800px;
      }
  </style>
</head>
<body>
<div id="map"></div>
<script>
  // https://leafletjs.com/examples/quick-start/
  const map = L.map("map").setView([51.505, -0.09], 13);

  L.tileLayer("https://tile.openstreetmap.org/{z}/{x}/{y}.png", {
    maxZoom: 19,
    attribution: "&copy; <a href=\"http://www.openstreetmap.org/copyright\">OpenStreetMap</a>"
  }).addTo(map);
</script>
</body>
</html>
```

File: lesson16.md (hunk 1 above)
Comment (olof-nord) [reply lesson16.md]: Said and done!

**Comment by olof-nord** [comment]
Merging now to have the code in place for the start of the lesson later

**Pull request merged**